sw/device/tests/rstmgr_cpu_info_test.c

```diff
@@ -24,176 +23,253 @@ OTTF_DEFINE_TEST_CONFIG();
  * RSTMGR CPU INFO TEST
  *  This test creates a double fault by accessing a register with
  *  a non-existing address.
- *  After the double fault, the dut gets reset by the watch dog bite,
+ *  After a single fault and then a double fault,
+ *  the dut gets reset by the watch dog bite,
  *  and the test collects / checks the cpu_info from the rstmgr.
  */
 
-// non existing address
-#define kIllegalAddr1 0x4041FFF0u
-#define kIllegalAddr2 0x40003618u
-#define kSkipComp 0x12345678u
+// Unmapped Addresses
+static const uint8_t *kIllegalAddr0 = (uint8_t *)0xA0A0DEAFu;
```

Comment:
These can fall in the translated address window (see [here](https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/sw/autogen/top_earlgrey_memory.ld;drc=0ea3c9ccf15917b9e61bfdef6c0637c5c2a42cf0;l=16)) depending on how we execute this test. Can you change `0xa.......` to `0xf.......`?

Reply:
Good spot, thank you. All changed.